src/main/java/dukeException/DukeException.java

```diff
@@ -0,0 +1,5 @@
+package dukeException;
+
+public class DukeException extends Throwable {
```

Comment:
Try not to leave your exceptions empty. Perhaps you could have the corresponding error message and a method to print/return the error message?